### ISSUE TITLE
Labels being recorded but not on bsky

### ISSUE THREAD

**Issue by cmho** [skyware-js/labeler#2]
Hi--I have a slightly odd question which may not be an issue per se. (Or at least, the issue may be problem-between-chair-and-keyboard.)

I've gotten a labeler set up, the labeling requests are definitely hitting my server and the server is responding (using most recent version of Caddy), the labels are being written to my sqlite database, and the promise with the bsky server is resolving such that it gives a label data object and says the labels are applied... but for some reason the labels don't show up on accounts liking the posts. I'm a little stymied; it's not throwing any errors whatsoever when I look at the logs.

My code repo is here: https://tildegit.org/smallbird/hiresky (in large part forked from pronouns.adorable.mom) if that helps.

**Comment by cmho** [comment]
oh, thanks for the quick reply! I can give that a shot once i'm off of work, and also glad to see that there's a discord because i have so many questions about stupid things i want to do with the bsky api

**Comment by cmho** [comment]
Update: Okay, it seems that while it *is* definitely running on both https and wss and I updated the label declarations (and those propagated to bsky.app), the labels still aren't appearing. I joined the bsky api touchers discord, though, so hopefully that'll turn up more ideas! Will update if I find a solution.

**Comment by coolroman** [comment]
@cmho Were you able to find a solution? I have the same problem.

**Comment by futurGH** [comment]
`npx @skyware/labeler label edit` without any changes will have the same effect without needing to redefine label values!

**Comment by coolroman** [comment]
> `npx @skyware/labeler label edit` without any changes will have the same effect without needing to redefine label values!

In my case it didn't work (I tried edit/add/delete label). So one can try edit label first and if it doesn't help do "clear".

**Comment by futurGH** [comment]
Interesting, I'll have to look into that

**Comment by cmho** [comment]
Unfortunately, doing the clear/setup rotation doesn't seem to work for me 😔

**Comment by ajnam12** [comment]
I am also running into the same issue and still haven't been able to solve it.

**Comment by ImLunaHey** [comment]
having the same issue. 

here's a post with a label https://bsky.app/profile/test-labeller-123.bsky.social/post/3lgnr2godjc2r

im subbed to the labeller on my main account and yet i dont see the label on the post.


<details><summary><code>getPostThread</code></summary>

```json
{
  "thread": {
    "$type": "app.bsky.feed.defs#threadViewPost",
    "post": {
      "uri": "at://did:plc:gclep67kb2bifr3praijwoun/app.bsky.feed.post/3lgnr2godjc2r",
      "cid": "bafyreibdx3gyp4gh55tb7dy4gk3d3z5wdeljfgrst63nkk6e5eolgrjh24",
      "author": {
        "did": "did:plc:gclep67kb2bifr3praijwoun",
        "handle": "test-labeller-123.bsky.social",
        "displayName": "",
        "avatar": "https://cdn.bsky.app/img/avatar/plain/did:plc:gclep67kb2bifr3praijwoun/bafkreif6gcheendfcwnveje2kskz7zj2o73nzlcdf6y6bn5yninh52o4zm@jpeg",
        "associated": { "labeler": true },
        "viewer": { "muted": false, "blockedBy": false },
        "labels": [],
        "createdAt": "2025-01-26T06:16:46.443Z"
      },
      "record": {
        "$type": "app.bsky.feed.post",
        "createdAt": "2025-01-26T15:38:36.226Z",
        "embed": {
          "$type": "app.bsky.embed.images",
          "images": [
            {
              "alt": "",
              "aspectRatio": { "height": 664, "width": 524 },
              "image": {
                "$type": "blob",
                "ref": {
                  "$link": "bafkreiaegr7bmir2wqr4buz6eyj5jogo3r477q6jtt4fcv2gzic6vg4ko4"
                },
                "mimeType": "image/jpeg",
                "size": 10215
              }
            }
          ]
        },
        "langs": ["en"],
        "text": "test"
      },
      "embed": {
        "$type": "app.bsky.embed.images#view",
        "images": [
          {
            "thumb": "https://cdn.bsky.app/img/feed_thumbnail/plain/did:plc:gclep67kb2bifr3praijwoun/bafkreiaegr7bmir2wqr4buz6eyj5jogo3r477q6jtt4fcv2gzic6vg4ko4@jpeg",
            "fullsize": "https://cdn.bsky.app/img/feed_fullsize/plain/did:plc:gclep67kb2bifr3praijwoun/bafkreiaegr7bmir2wqr4buz6eyj5jogo3r477q6jtt4fcv2gzic6vg4ko4@jpeg",
            "alt": "",
            "aspectRatio": { "height": 664, "width": 524 }
          }
        ]
      },
      "replyCount": 0,
      "repostCount": 0,
      "likeCount": 0,
      "quoteCount": 0,
      "indexedAt": "2025-01-26T15:38:37.454Z",
      "viewer": { "threadMuted": false, "embeddingDisabled": false },
      "labels": [
        {
          "cid": "bafyreibdx3gyp4gh55tb7dy4gk3d3z5wdeljfgrst63nkk6e5eolgrjh24",
          "cts": "2025-01-26T15:38:37.561Z",
          "neg": false,
          "src": "did:plc:gclep67kb2bifr3praijwoun",
          "uri": "at://did:plc:gclep67kb2bifr3praijwoun/app.bsky.feed.post/3lgnr2godjc2r",
          "val": "no-alt-text",
          "ver": 1
        }
      ]
    },
    "replies": [],
    "threadContext": {}
  }
}
```
</details>

**Comment by futurGH** [comment]
Did you try running the recreate command?

**Comment by ImLunaHey** [comment]
might be worth suggesting people uses `npx @skyware/labeler@latest` as that command must have been added recently and wasnt showing for me. 😅 

tried that just then, no luck so far. (not sure if i have to wait)

**Comment by ImLunaHey** [comment]
ive even tried something simple and i still cant see the label even though it shows in `getPostThread`.

```ts
labelerServer.createLabel({
  uri: 'at://imlunahey.com/app.bsky.feed.post/3lgyti2673c2u',
  neg: false,
  val: 'no-alt-text',
});
```

**Comment by futurGH** [comment]
It's not really a matter of simple/not — if it still isn't working after recreating the labeler record, all you can really do is start over on a new did, try not to wait too long between setting up the service record and having the labeler running, and hope the AppView picks it up this time.

**Comment by ImLunaHey** [comment]
@futurGH sorry but thats not a solution thats a "idk what the issue is" response.

**Comment by futurGH** [comment]
Yeah? The label ingester isn't open source so that's the most any of us who don't work at Bluesky can figure out

**Comment by ImLunaHey** [comment]
so.. we ask them. i really dont get either of these replies.

**Comment by ImLunaHey** [comment]
ive messaged some internal people at bluesky and will update this when i have more information.

**Comment by futurGH** [comment]
Yeah go ahead, I'm just generally not going to recommend people go pester Jaz on a public repo

**Comment by ImLunaHey** [comment]
sorry but its not pestering to ask questions when we dont have docs. not sure where you got that idea.

**Comment by futurGH** [comment]
Not saying you are, just that it'd be a nuisance to have however many people asking them to clear the cache, so I'm not going to encourage it

**Comment by cmho** [comment]
circling back—does the issue seem to be taking too long between creating an account and setting up the labeler?
